custom/icds/data_management/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from corehq.const import USER_DATETIME_FORMAT
+from custom.icds.data_management.const import DATA_MANAGEMENT_TASKS
+from custom.icds.data_management.models import DataManagementRequest
+
+
+class DataManagementRequestSerializer(serializers.ModelSerializer):
+    class Meta(object):
+        model = DataManagementRequest
+        fields = ['db_alias', 'start_date', 'end_date', 'domain',
+                  'initiated_by', 'error', 'status']
+
+    def to_representation(self, instance):
```

Comment:
A more standard way of doing this would be to use the `format` param of the [DateTimeField](https://www.django-rest-framework.org/api-guide/fields/#datetimefield)

For the others you can make them properties on the `DataManagementRequest` class and then refer to them as normal fields in the serializer.

Reply:
Thanks @snopoke ,
Updated to use DateTimeField

My main intention to use serialiser was to avoid putting such representation related logic on the model. So retaining logic for other fields here only.

Reply:
Adding a whole new package and class seems worse than adding a few properties to the class. But not a blocker.